src/components/profile/ProfileService.js

```diff
@@ -331,9 +361,9 @@ goog.require('ga_urlutils_service');
 
           //Icon for elevation up
           group.append('text')
-              .attr('class', 'ga-profile-icon')
+              .attr('font-family', 'FontAwesome')
```

Comment:
Because I added `font-size: 0.9em;` at the class `ga-profile-icon`
and I use this class for all the profile icons except for the icons elev-up and elev-down,
which need to have `font-size: 1.5em;`